integration/kafka/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I think we can do a better job of packing here. Essentially we don't need a fat jar, right? If possible, would you mind to follow how do we package in the HDFS connector?